its/src/test/java/com/sonar/maven/it/suite/BootstrapTest.java

```diff
@@ -61,6 +60,93 @@ void test_unsupported_platform() {
     }
   }
 
+  @Test
+  void test_bootstrapping_with_sonar_skip_in_pom_xml() {
+    MavenBuild build = MavenBuild.create(ItUtils.locateProjectPom("maven/bootstrap-small-project"))
+      .setProperty("sonar.login", ORCHESTRATOR.getDefaultAdminToken())
+      .setProperty("sonar.host.url", ORCHESTRATOR.getServer().getUrl())
+      // activate in the pom.xml <properties><sonar.skip>true</sonar.skip></properties>
+      .addArguments("-Ptest-sonar-skip")
+      .setGoals(sonarGoal());
+    BuildResult result = executeBuildAndValidateWithoutCE(build);
+    assertThat(result.getLogs())
+      .contains("Skipping analysis")
+      .doesNotContain("JRE provisioning:", "Communicating with SonarQube Server", "Starting SonarScanner Engine");
```

Comment:
We can make `"Skipping analysis"` and `"JRE provisioning:", "Communicating with SonarQube Server", "Starting SonarScanner Engine"`, two different constants to ease the work if something changes